### PR TITLE
Fix: scheduler with missing credit cards

### PR DESCRIPTION
**Pull request by caillou**
Todo:

- [x] Fix `membership_owner_autopay_notice`
- [x] Fix `membership_owner_autopay`

### CODE REVIEW

**Comment by caillou** [comment]
@tpreusse Would you by any chance know, how I can locally test this code?

File: packages/republik-crowdfundings/lib/scheduler/owners.js [hunk 1]
```diff
@@ -291,3 +294,34 @@ module.exports = {
...
+  const pgdb = context.pgdb
+  const defaultPaymentSource = getDefaultPaymentSource(user, pgdb)
+
+  if (!defaultPaymentSource) {
+    await setAutoPayToFalse(user, pgdb)
+    return
+  }
+
+  user = { ...user, defaultPaymentSource }
+  return mailings(user, bucket, context)
```
Comment (tpreusse) [inline packages/republik-crowdfundings/lib/scheduler/owners.js:308]: (card is expected on `user.autoPay.card` by downstream functions not `user.defaultPaymentSource`, but no longer relevant if we move to `suggest`)

**Review by tpreusse — CHANGES_REQUESTED** [review]

**Review by tpreusse — COMMENTED** [review]
One more thing ;-)

**Review by tpreusse — APPROVED** [review]

File: packages/republik-crowdfundings/lib/AutoPay.js [hunk 2]
```diff
@@ -73,22 +73,9 @@ const suggest = async (membershipId, pgdb) => {
 
   const rewardId = membershipPledgeOption.packageOption.rewardId
 
-  // Find pledge payments
-  const pledgePayments = await pgdb.public.pledgePayments.find({ pledgeId: pledge.id })
+  const defaultPaymentSource = getDefaultPaymentSource(membership.userId, pgdb)
```
Comment (tpreusse) [inline packages/republik-crowdfundings/lib/AutoPay.js:76]: an await is missing 🙈

**Pull request merged**